samples/custom_dispatch/cpu/mlp_plugin/CMakeLists.txt

```diff
@@ -108,4 +113,65 @@ iree_lit_test_suite(
     "hostonly"
 )
 
+# Append the includes used for this file to the pdll_compilation_commands
+file(APPEND ${CMAKE_BINARY_DIR}/pdll_compile_commands.yml
+  "--- !FileInfo:\n"
+  "  filepath: \"${CMAKE_CURRENT_SOURCE_DIR}/mlp_all_specs.pdll\"\n"
+  "  includes: \"${TORCH_MLIR_ROOT_DIR}/include;${CMAKE_CURRENT_SOURCE_DIR};${MLIR_INCLUDE_DIRS}\"\n"
```

Comment:
I'm quite skeptical of the references to `TORCH_MLIR_ROOT_DIR` in this PR. What is that used for? We should be defining a stable interface on files distributed with releases, and I can't easily see what this PDLL code actually depends on.

Reply:
removed